app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue

```diff
@@ -352,6 +353,10 @@ export default {
       this.$emit('assign-label', [label.title], [this.chat.id]);
       this.closeContextMenu();
     },
+    async onAssignTeam(team) {
+      this.$emit('assign-team', team, [this.chat.id]);
```

Comment:
Why do we need to send this as an array? We are not using the bulkActions API here, right?

Reply:
Fixed